ShitFood.Web/js/get-restaurants.js

```diff
@@ -0,0 +1,35 @@
+var axios = require('axios');
+
+function getRestaurants() {
+  var cutleryIcon = L.icon({
+    iconUrl: '../img/markers/cutlery.svg',
```

Comment:
This needs to be './img/markers/cutlery.svg' to work

Reply:
Nice